rfcs/convergence/changelog-guidelines.md

```diff
@@ -0,0 +1,130 @@
+
+## Summary
+
+This RFC proposes a set of guidelines when creating changefiles during a PR. These changefiles are directly translated
+(in most cases) to our changelog which is important for users to see what changed between releases of Fluent. Therefore
+this RFC proposes the following requirements for the changelog:
+
+- Changes that do not affect published code should not be in the changelog
+- Clear labels for change types
+- The detail of changes should be descriptive first and short second
+- Messages should be written for changes specific to the package, not the PR
+
+The changefiles also directly influence how packages are bumped and released in the release pipeline.
+
+## Problem statement
+
+The current v9 changelog contains (or contained before removal) the following bad patterns that should not be in the changelog:
+
+### Changelog entries that do not affect published code
+
+Changes such as:
+
+- stories are updated
+- dev dependencies are updated
+- tests were added/updated
+- scripts were modified
+
+Should not be included in the changelog. Generally when these kinds of changes are in the changelog they should be
+removed as soon as they are noticed.
+
+It's also important that these kinds of change entries **should not bump upstream packages**. By bumping upstream
+packages, we create new releases along the whole chain of dependencies where in fact nothing has actually changed
+for partners. Even in the context of lockstep versioning these kinds of changes should not result in version bumps.
+
+### Changelog entries that are focused on the PR
+
+Quite often, a PR will address/fix an issue that spans several packages. While it's good practice to keep PRs as
+small as possible, it is still possible that mutliple changes in different packages that need to be modified. This
+can also be acceptable when an internal API is modified in a non breaking way, but several consumers need to be updated
+at the same time.
+
+It is very easy to write a single changefile message during PR creating and apply it to all the changed packages in
+the PR. This makes the change harder to understand since the contents might only apply vaguely, or not at all to the package.
+
+A simple example would be `Fix dependencies`, this kind of generic description can be applied to a PR when linked to
+an issue, but has an incredibly vague meaning with applied to the changelog of a package.
+
+## Detailed design or proposal
+
+### Semantic labels for changefile entries
+
+Each changefile entry should have a message that has a semantic label that is inspired by [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/).
+This RFC proposes four labels in the following form:
+
+```
+feat:  This label communicates that a feature was added to the package
+fix:   This label communicates that a fix was applied to the package
+chore: This label communicates that changes which should not have an affect on th user were applied
```

Comment:
Spelling nit. Also I personally dislike the "chore" naming and this notation in general, but it appears I'm thoroughly outnumbered in the open source community.
```suggestion
chore: This label communicates that changes which should not have an affect on the user were applied
```